packages/fast-form-generator-react/src/form/form-item.array.style.ts

```diff
@@ -31,8 +33,25 @@ const styles: ComponentStyles<IFormItemArrayClassNameContract, {}> = {
         ...applyAriaHiddenStyles(),
         ...applyPopupMenuStyles(),
         "& li": {
```

Comment:
Do we still need to target the `li` if we have no styles bound to it and the only styles in it are nested?

Reply:
Refactored and removed